test_scripts/RC/InteriorVehicleData_cache/common_interiorVDcache.lua

```diff
@@ -0,0 +1,331 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local commonRC = require('test_scripts/RC/commonRC')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local json = require("modules/json")
+local utils = require("user_modules/utils")
+
+--[[ Module ]]
+local m = actions
+m.cloneTable = utils.cloneTable
+m.jsonFileToTable = utils.jsonFileToTable
+m.tableToJsonFile = utils.tableToJsonFile
+m.wait = utils.wait
+m.tableToString = utils.tableToString
+
+local preloadedPT = commonFunctions:read_parameter_from_smart_device_link_ini("PreloadedPT")
+m.preconditionOrigin = m.preconditions
+local postconditionOrigin = m.postconditions
+
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+config.application2.registerAppInterfaceParams.isMediaApplication = false
+
+m.modules = { "CLIMATE", "RADIO" }
```

Comment:
Note these modules should be extended with the additional RC modules after they have been reviewed and merged.

Reply:
@JackLivio, thanks for the note! 
For now all modules are merged into develop, so code with `Interior Vehicle Data Subscription Management` will be expanded with new modules and also i will add one more commit with expanded scripts for all new modules.
For now the expanding of the scripts is in progress.